### DISCUSSION TITLE
Fajar Anugrah | ba@p

### DISCUSSION THREAD

**Discussion by hargunarajaf** [onlyphantom/llm-python#22] [Practicum: Generative AI]
https://colab.research.google.com/drive/1dtM90n480l5JJxB1ygAuDImeqaNaQAD-#scrollTo=TWLSMUJP0BjH

**Comment by onlyphantom** [comment]
Hey Fajar, can you make the notebook public? Can't access the link for grading! :)

**Reply by hargunarajaf** [reply]
done~

**Comment by onlyphantom** [comment]
I'll mark this as Passed, but the following prompt should ideally not use any hard coded value! If you build an AI system in real life, you expect the user to be able to use your AI any day, and that the date should reflect that actual date!

```
            "today's date is 31 july 2024. answer the queries. if not prompted, assume top company is 1",
```

could be improved to call a function returning the actual date at query-time!

You can see an example implementation here:
https://docs.sectors.app/recipes/generative-ai-python/02-tool-use

Thank you for being an excellent participant!
